$NetBSD: patch-ac,v 1.4 2001/05/03 22:03:31 dmcmahill Exp $

a bug with -ggdb on powerpc was tickled by some of the code for ucsim
so don't use it on that platform.

--- configure.orig	Sat Sep 30 17:37:31 2000
+++ configure	Thu May  3 15:06:17 2001
@@ -1084,5 +1084,5 @@
 echo $ac_n "checking for a BSD compatible install""... $ac_c" 1>&6
 echo "configure:1086: checking for a BSD compatible install" >&5
-if test -z "$INSTALL"; then
+if true; then
 if eval "test \"`echo '$''{'ac_cv_path_install'+set}'`\" = set"; then
   echo $ac_n "(cached) $ac_c" 1>&6
@@ -2324,5 +2324,5 @@
 EOF
 $CC -v -ggdb -c _test_.c 1>&5 2>&5
-if test "$?" = "0"; then
+if test "$?" = "0" -a "$MACHINE_ARCH" != "powerpc" ; then
   sdcc_cv_CCggdb="yes"
 else
